Extension/ASPNET-MVC-Extension/Project-Migration.md

```diff
@@ -40,18 +40,18 @@ The following steps help you migrate from one version to another version of your
    **Option 1:**  
    Click **Syncfusion Menu** and choose **Essential Studio for ASP.NET MVC (EJ1) > Migrate Project…** in **Visual Studio**.
 
-   ![Syncfusion Essential JS 1 ASP.NET MVC Project Migration via Syncfusion menu](Migrate-Syncfusion-Project_images/SyncfusionMenu_ProjectMigration_img.png)
+   ![Syncfusion Essential JS 1 ASP.NET MVC Extension in Project Migration via Syncfusion menu](migrate-syncfusion-project-images/aspnet-mvc-extension-for-project-migration-in-syncfusion-menu.png)
 
    N> In Visual Studio 2019, Syncfusion menu available under Extension in Visual Studio menu.
 
    **Option 2:**  
    Right-click the **Syncfusion ASP.NET MVC Application** from Solution Explorer and select **Syncfusion Essential JS 1**. Choose **Migrate the Essential JS 1 Project to Another Version...**
 
-   ![Syncfusion Essential JS 1 ASP.NET MVC Project Migration add-in](Migrate-Syncfusion-Project_images/ProjectMigration_img.png)
+   ![Syncfusion Essential JS 1 ASP.NET MVC Extension in Project Migration add-in](migrate-syncfusion-project-images/aspnet-mvc-extension-in-project-migration-add-in.png)
```

Comment:
check image path in all places
<img width="225" alt="image" src="https://github.com/syncfusion-content/extension-docs/assets/61148953/2ade9cf8-104e-4e62-825d-f8b30b9e04b3">

<img width="572" alt="image" src="https://github.com/syncfusion-content/extension-docs/assets/61148953/ba6b8d7d-5668-46e8-9686-32c696971341">

Reply:
Resolved the issues